src/DynamoCore/BuiltInAndOperators/BuiltIn.xml

```diff
@@ -76,6 +76,7 @@
       <summary>Returns the items from the list that generate the maximum values for the function supplied as the key projector</summary>
       <param name="list">list of values</param>
       <param name="keyProjector">function applied to the list items</param>
+      <returns name="maxItem">maximum item in list using keyProjector</returns>
```

Comment:
same as with the `minItem` might make sense to call it `maximumItem`. Personally fine with both so i guess its a question of how much we want to spell things out. Looking at the other changes we done like changing `bool` to `boolean`, this might also need to be more explicit.

Reply:
addressed in PR 
4df71eb